internal/pkg/cli/storage_init_test.go

```diff
@@ -161,6 +162,15 @@ func TestStorageInitOpts_Validate(t *testing.T) {
 			inNoSort:      true,
 			wantedErr:     fmt.Errorf("validate LSI configuration: cannot specify --no-sort and --lsi options at once"),
 		},
+		"invalid database engine type": {
+			inAppName: "meow",
```

Comment:
🐈

Reply:
Meow is my profile pic!! My sweet doggo!

Reply:
Oh, that's right! Duh! 🐕